contrib/natsio/pom.xml

```diff
@@ -0,0 +1,58 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
It would be good to follow the model set out in the [contrib join library](https://github.com/GoogleCloudPlatform/DataflowJavaSDK/blob/master/contrib/join-library/pom.xml). Specifically

- Artifact naming
- License information
- Compiler source and target versions (1.7, probably)
- optionally, Checkstyle

- review dependencies